lib/client.ts

```diff
@@ -0,0 +1,42 @@
+import { Kafka } from 'kafkajs';
+
+export const produceAndConsumeMessage = async function (
```

Comment:
Would it make sense at this point of time to split this into two separate functions?

Reply:
I think to speed up the process we could leave it together and later split it into 2 more atomic methods

Reply:
Agree with @asanzgom comment